devlib/instrument/daq.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, target, resistor_values,  # pylint: disable=R0914
                                                  resistor_values=resistor_values,
                                                  channel_map=channel_map,
                                                  labels=labels)
+        self.sample_rate_hz = sampling_rate
```

Comment:
OK, I actually thought about this but then thought I should not break backwards compatibility. Don't know how many people use DAQs.